ts/Core/Chart/ChartDefaults.ts

```diff
@@ -992,13 +994,65 @@ const ChartDefaults: ChartOptions = {
      * However, `zoomBySingleTouch` will interfere with touch-dragging the
      * chart to read the tooltip. And especially when vertical zooming is
      * enabled, it will make it hard to scroll vertically on the page.
-     * @since 9.0.0
+     * @since      9.0.0
      * @sample     highcharts/chart/zoombysingletouch
      *             Zoom by single touch enabled, with buttons to toggle
      * @product    highcharts highstock gantt
+     * @deprecated
      */
     zoomBySingleTouch: false,
 
+    /**
+     * Chart zooming options.
+     * @since       next
+     */
+    zooming: {
```

Comment:
In addition to these three zoom properties, we also have `zoomKey` which I believe belongs here.

Reply:
Added zoomKey, what do you think about adding `chart.resetZoomButton` to zooming options, in the future?

Reply:
Yes, I think `resetZoomButton` also belongs here. Then as `zooming.resetButton`.